src/components/TextAreaInput/TextAreaInput.tsx

```diff
@@ -15,13 +14,14 @@ const TextAreaInput: React.FC<TextAreaInputProps> = ({ name, placeholder }) => {
   const errorMessage = errors[name]?.message as string | undefined;
 
   return (
-    <div className="text-area">
+    <div className="flex flex-col items-center">
       <textarea
-        className="text-area-input"
+        className="w-full h-[167px] rounded-[12px] p-3.5 bg-[#E9E8D5] mb-2.5 box-border 
+        text-base font-roboto leading-[19.2px] resize-none"
         {...register(name, { required: "This field is required" })}
         placeholder={placeholder}
       />
-      {errorMessage && <p className="text-area-error">{errorMessage}</p>}
+      {errorMessage && <p className="text-[#CF534D]">{errorMessage}</p>}
```

Comment:
Please discuss with Joshua the color, styling and tags used for his validation message. 
On his pull request for the phone and email he used the color of red-600 for the invalid message and he also used a span tag.
You also don't want the validation message to appear under the input because for the name field, there is no room under the input for the validation message.